octobot_trading/personal_data/orders/order.py

```diff
@@ -301,7 +327,16 @@ async def on_filled(self):
         """
         Filling complete callback
         """
-        # nothing to do by default
+        await self._trigger_chained_orders()
+
+    async def _trigger_chained_orders(self):
+        logger = logging.get_logger(self.get_logger_name())
+        for index, wrapped_order in enumerate(self.chained_orders):
+            if wrapped_order.should_be_created():
+                logger.debug(f"Creating chained order {index + 1}/{len(self.chained_orders)}")
+                await wrapped_order.create_order()
+            else:
+                logger.debug(f"Skipping cancelled chained order {index + 1}/{len(self.chained_orders)}")
```

Comment:
Is it WIP logs?

Reply:
It's to keep track on order that are created this way in debug logs (not WIP)